src/java.desktop/macosx/classes/com/apple/laf/AquaLabelUI.java

```diff
@@ -25,14 +25,19 @@
 
 package com.apple.laf;
 
-import java.awt.*;
+import java.awt.Color;
+import java.awt.Graphics;
 
-import javax.swing.*;
-import javax.swing.plaf.*;
-import javax.swing.plaf.basic.*;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+
+import javax.swing.plaf.ComponentUI;
+import javax.swing.plaf.UIResource;
+
+import javax.swing.plaf.basic.BasicLabelUI;
 
-import sun.swing.SwingUtilities2;
 import sun.swing.MnemonicHandler;
+import sun.swing.SwingUtilities2;
 
 import com.apple.laf.AquaUtils.RecyclableSingleton;
 import com.apple.laf.AquaUtils.RecyclableSingletonFromDefaultConstructor;
```

Comment:
I'd sort them alphabetically too: `sun.swing` follows `com.apple.laf`. I'd not add a blank line between these imports.